kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -533,7 +533,17 @@ private Optional<Long> getStartCheckpointVersion(Engine engine, Optional<Long> v
                       });
             })
         .orElseGet(
-            () -> new Checkpointer(logPath).readLastCheckpointFile(engine).map(x -> x.version));
+            () -> {
```

Comment:
@allisonport-db: Is there any reason why `timers` like this are not added to the SnapshotReport?

Reply:
No reason; we should add new metrics iteratively as we see fit!